### PR TITLE
Apply nl2br on long text fields in SAP mission view

### PR DESCRIPTION
**Pull request by polybuildr**
Resolves T140.

### CODE REVIEW

**Comment by hharchani** [comment]
LGTM.

**Comment by polybuildr** [comment]
But you won't merge? :P

File: src/app/views/sap/mission.php [hunk 1]
```diff
@@ -50,9 +50,9 @@
         <?php foreach ($tasks as $i=>$task): ?>
             <h3>Task <?= $i + 1 ?></h3>
             <?php if (isset($task['submission']) && $task['submission']['done'] !== 2): ?>
-                <p><?= htmlspecialchars($task['description']) ?></p>
+                <p><?= nl2br(htmlspecialchars($task['description'])) ?></p>
                 <?php if ($task['has_text_answer']): ?>
-                    <textarea disabled><?= htmlspecialchars($task['submission']['answer']) ?></textarea>
+                    <textarea disabled><?= nl2br(htmlspecialchars($task['submission']['answer'])) ?></textarea>
```
Comment (nisargjhaveri) [inline src/app/views/sap/mission.php]: I don't think nl2br here is appropriate.

**Comment by polybuildr** [comment]
I hadn't realised that this hasn't been merged yet.

**Comment by nisargjhaveri** [comment]
Wow!

**Comment by nisargjhaveri** [comment]
Failing build.

**Comment by polybuildr** [comment]
> Could not find .travis.yml, using standard configuration.

Weird. Re-running build.

**Comment by polybuildr** [comment]
Re-running the build does not help. I filed https://github.com/travis-ci/travis-ci/issues/5389. Meanwhile, I'm going to try amending the commit and pushing for no reason.

**Comment by polybuildr** [comment]
And as far as I can see, the issue has been miraculously resolved. :P

**Comment by nisargjhaveri** [comment]
Looks good.

**Pull request merged**